lake/index/rule.go

```diff
@@ -65,10 +64,6 @@ func NewTypeRule(name string, typ zed.Type) *TypeRule {
 }
 
 func NewAggRule(name, prog string) (*AggRule, error) {
-	// make sure it compiles
-	if _, err := compiler.ParseOp(prog); err != nil {
-		return nil, err
-	}
```

Comment:
Are you sure you want to get rid of this?

If you do, consider removing the error return parameter.

Reply:
Ok I put it back.  Needed to pass in the a runtime.Compiler because lake packages can't import compiler.